src/providers/wms/qgswmscapabilities.cpp

```diff
@@ -783,6 +783,34 @@ void QgsWmsCapabilities::parseLegendUrl( QDomElement const &e, QgsWmsLegendUrlPr
   QgsDebugMsg( QStringLiteral( "exiting." ) );
 }
 
+void QgsWmsCapabilities::parseMetadataUrl( QDomElement const &e, QgsWmsMetadataUrlProperty &metadataUrlProperty )
+{
+
+  QDomNode n1 = e.firstChild();
+  while ( !n1.isNull() )
+  {
+    QDomElement e1 = n1.toElement(); // try to convert the node to an element.
+    if ( !e1.isNull() )
+    {
+      QString tagName = e1.tagName();
+      if ( tagName.startsWith( QLatin1String( "wms:" ) ) )
+        tagName = tagName.mid( 4 );
+
+      if ( tagName == QLatin1String( "Format" ) )
+      {
+        metadataUrlProperty.format = e1.text();
+      }
+      else if ( tagName == QLatin1String( "OnlineResource" ) )
+      {
+        parseOnlineResource( e1, metadataUrlProperty.onlineResource );
+      }
+    }
+    n1 = n1.nextSibling();
+  }
+
+  QgsDebugMsg( QStringLiteral( "exiting." ) );
```

Comment:
Is this here on purpose?

Reply:
> (btw - I realise the above review looks petty, but the project is quite strict on formatting and naming standards, and it's worth getting familiar with them upfront as it'll speed up all future reviews!)

Sure, seems the whole class didn't follow the project standards from first, will have to change it all.

Reply:
I don't know, it is a convention used in similar functions. Do you think we should remove it?

Reply:
Yes, please.
If there is a good reason to add some here and there, there is no objection. But it shouldn't be done by convention, because it's a lot of debug noise that's added.

Reply:
Got it